### DISCUSSION TITLE
jSystemThemeDetector

### DISCUSSION THREAD

**Discussion by Dansoftowner** [0x4a616e/jfa#1] [Show and tell]
Hi!

I made a [library](https://github.com/Dansoftowner/jSystemThemeDetector) that can be used for detecting
that whether the operating system uses a dark theme or not.

In case of MacOS, I used this JFA library for the native access.
 
Thank you for creating this project, Sir!

**Comment by 0x4a616e** [comment]
Nice one! Maybe we can add `NSColorChangesObserver`, `NSDistributedNotificationCenter` to the API interfaces. `NSUserDefaults` should already be there, so you might have used `NSUserDefaults.standardUserDefaults().objectForKey(NSUserDefaults.AppleInterfaceStyle)` which is a bit simpler than working with the Foundation class.
